src/pages/CommandView/CommandJobForm.tsx

```diff
@@ -0,0 +1,190 @@
+  const onSubmit = (data: {[key: string]: unknown}) => {
+    if(!Object.keys(errors).length) {
+      let formData: FormData | undefined
+      let headers = {}
+      if(Object.hasOwn(data, 'multipart') && data['multipart']) {
+        const multipartData = data['multipart'] as {[key: string]: File}
+        delete data['multipart']
+        formData = new FormData()
+        formData.append('request', JSON.stringify(data))
+        Object.keys(multipartData).forEach((key) => {
+          if(formData) formData.append(key, multipartData[key] as File)
+        })
+        headers = { 'Content-type': 'multipart/form-data' }
+      }
+      const requestData: RequestTemplate = data as unknown as RequestTemplate
+      let path = '/api/v1/requests'
+      let method: Method = 'post'
+      let operation: PatchOperation | undefined
+      if(isJob) {
+        if(job){
+          method = 'patch'
+          path = `/api/v1/jobs/${job.id}`
+          operation = { operations: [
+            {
+              operation: 'update',
+              path: '/job',
+              value: data as unknown as Job
+            }
+          ]}
+        } else {
+          path = '/api/v1/jobs'
+        }
+      }
+      const config: AxiosRequestConfig<FormData | PatchOperation | RequestTemplate> = {
+        url: path,
+        method: method,
+        data: formData || operation || requestData,
+        headers: headers,
+        withCredentials: authEnabled,
+      }
+
+      execute(config)
+        .then((response: AxiosResponse<Request | Job>) => {
+          let path = '/requests/'
+          if(isJob) path = '/jobs/'
+          navigate(path + response.data.id)
+        })
+        .catch((error: AxiosError) => {
+          raiseError(JSON.stringify(error.toJSON()))
+        })
+    }
+  }
+
+  const formReset = () => {
+    if(isJob) {
+      if(showJobFields){
+        const tempDefaultValues = defaultValues
+        delete tempDefaultValues['request_template']
+        reset(formValues => ({
+          ...formValues,
+          ...tempDefaultValues,
+        }))
+      } else resetField('request_template', { defaultValue: defaultValues.request_template })
+
+    } else (
+      reset(defaultValues)
+    )
+  }
+
+  return (
+    <>
+      <FormProvider {...methods} >
+        <Grid columns={5} spacing={1} py={1} alignItems="start" justifyContent="space-between" container>
+          <Grid minWidth="300px" xs={3} key="form" item>
+            <form onSubmit={handleSubmit(onSubmit)} >
+              <Stack rowGap={2} >
+                {!showJobFields ?
+                  <CommandFormFields parentKey={isJob ? 'request_template' : ''} parameters={command.parameters} instances={system.instances} />
+                  :
+                  <JobSettingFormFields />
+                }
+                <Stack direction="row" spacing={1} >
+                  <Button sx={{width: '150px'}} type="button" onClick={() => formReset()} variant="contained" >Reset</Button>
```

Comment:
I will make a ticket to convert the param components  to use the generic form components since that should resolve the reset issue. I already did it for the component used by the `sleep`, so the field should clear on reset.